src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelEventArgs.cs

```diff
@@ -153,4 +159,142 @@ public OpenDocumentationLinkEventArgs(Uri link)
             IsRemoteResource = link.IsAbsoluteUri && !link.IsFile;
         }
     }
+
+    /// <summary>
+    /// A node documentation request compatible with the Dynamo RequestOpenDocumentationLink event,
+    /// such as the namespace of the node used to lookup the .md documentation file
+    /// and additional Node Info gathered from the NodeModel properties.
+    /// </summary>
+    public class OpenNodeAnnotationEventArgs : OpenDocumentationLinkEventArgs
+    {
+        public string Namespace { get; }
+        public string Type { get; }
+        public string Description { get; }
+        public string Category { get; }
+        public List<string> InputNames { get; private set; }
```

Comment:
is there a reason these are actually `List<T>` in their public signature? If you can use IEnumerable<T> thats more flexible - especially if the underlying implementation doesn't need to be a list.